draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -1190,7 +1190,11 @@ handshake is as follows:
 When a host wants to close an existing subflow but not the whole MP-DCCP
 connection, it initiates the regular DCCP connection termination procedure 
 as described in {{RFC4340}}, i.e., it sends a DCCP-Close/DCCP-Reset on the subflow. This
-may be preceded by a DCCP-CloseReq.
+may be preceded by a DCCP-CloseReq. In the event of an irregular termination of a subflow,
+e.g., during subflow establishment, it is RECOMMENDED to use an appropriate DCCP reset code as specified in
+Table 2 of {{RFC4340}}. This could be, for example, sending reset code 5 (Option Error) when an MP-DCCP
+option provides invalid data or reset code 9 (Too Busy) when the maximum number of maintainable paths
+is reached. Note that receiving a reset code 9 for secondary subflows SHOULD NOT impact already existing active subflows. These subflows will be terminated following the procedure defined in Section XX, when needed. 
```

Comment:
@boucadair, 

1. `Note that receiving a reset code 9 for secondary subflows SHOULD NOT impact already existing active subflows.`

Not sure if I get your intention here, as this should be obvious already. If this note is required from your view, why this has not to be mentioned for reset code 5?

2. `These subflows will be terminated following the procedure defined in Section XX, when needed.`

If a remote host is Too Busy I don't see that this requires a special procedure. Either the remote host can keep ongoing active connections or remote host can decide to close individual subflows as outlined in this paragraph or close the whole MP-DCCP connection using MP_CLOSE or MP_FASTCLOSE.

I propose to remove both sentences.